almalinux-devtoolset11/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM almalinux:8-minimal
+
+USER 0
+
+# Need shadow-utils for groupadd and useradd
+# TODO: "source scl_source enable gcc-toolset-11" only applies to current session
```

Comment:
I don't know how to solve this. Could add gcc binaries to PATH but I assume `source scl_source enable gcc-toolset-11` does more than that.